src/test/java/io/ParserTest.java

```diff
@@ -0,0 +1,18 @@
+package io;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest{
+    @Test
+    public void defaultStringTest() {
+        String TEST = "Test String 123/by 123312/from 1394";
```

Comment:
If you intend for your `TEST` variable to be const, consider using the keyword `final`

Reply:
Thank you! I missed this throughout the rest of the repo too. Good catch!